src/main/java/com/exadel/discountwebapp/discount/entity/Discount.java

```diff
@@ -0,0 +1,100 @@
+package com.exadel.discountwebapp.discount.entity;
+
+import com.exadel.discountwebapp.category.entity.Category;
+import com.exadel.discountwebapp.location.entity.Location;
+import com.exadel.discountwebapp.vendor.entity.Vendor;
+import lombok.*;
+import org.springframework.data.annotation.CreatedDate;
+import org.springframework.data.annotation.LastModifiedDate;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
+
+import javax.persistence.*;
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+import java.util.List;
+
+@Entity
+@Table(name = "discount")
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@EntityListeners(AuditingEntityListener.class)
+public class Discount {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @EqualsAndHashCode.Exclude
+    @Column(name = "dis_id")
+    private Long id;
+
+    @Column(name = "dis_title", length = 25, nullable = false)
+    private String title;
+
+    @ToString.Exclude
+    @EqualsAndHashCode.Exclude
+    @Column(name = "dis_short_description", length = 50, nullable = false)
+    private String shortDescription;
+
+    @ToString.Exclude
+    @EqualsAndHashCode.Exclude
+    @Column(name = "dis_description", length = 510, nullable = false)
+    private String description;
+
+    @ToString.Exclude
```

Comment:
I don't see any reason why we should exclude from ToString the following fields - imageUrl, description, shortDescription

Reply:
done